.vscode/settings.json

```diff
@@ -2,5 +2,11 @@
     "[python]": {
         "editor.defaultFormatter": "ms-python.black-formatter",
         "editor.formatOnSave": true
+    },
+    "python.pythonPath": "C:\\Users\\chand\\miniconda3\\envs\\wod\\python.exe"
+    {
+        "python.terminal.activateEnvironment": true,
+        "terminal.integrated.shell.windows": "C:\\Windows\\System32\\cmd.exe"
```

Comment:
Could we clear this as well? The python path is specific to your system. You may want to find another way to configure this for your machine.